app/forms/validators.py

```diff
@@ -128,39 +128,64 @@ def __call__(
         field: Union[DecimalFieldWithSeparator, IntegerFieldWithSeparator],
     ) -> None:
         value: Union[int, Decimal] = field.data
+        decimal_places = 0
+        if field.type == "DecimalFieldWithSeparator":
+            decimal_places = field.places
 
         if value is not None:
-            error_message = self.validate_minimum(value) or self.validate_maximum(value)
+            error_message = self.validate_minimum(
+                value, decimal_places
+            ) or self.validate_maximum(value, decimal_places)
             if error_message:
                 raise validators.ValidationError(error_message)
 
-    def validate_minimum(self, value: NumType) -> Optional[str]:
+    def validate_minimum(
+        self, value: NumType, decimal_places: int | None
```

Comment:
Minor, enforce keyword only args?. or at least for decimal places.

Reply:
Updated 👍